config/nav.yml

```diff
@@ -100,6 +100,20 @@ nav:
         - Architecture: serving/architecture.md
         - Request Flow: serving/request-flow.md
       - Resources:
+        - Services:
```

Comment:
What is the idea about moving this? I (personally) do not get what why "Resources" and "Developer Topics" are two chapters in the first place. Maybe merge them all to getter?

Reply:
This was an idea coming from the other PR. Having just revisions didnt make much sense, Services fall under the same category. I will take a look and see if we can have a better grouping but this could be something to handle in a different PR as well.